daemon/args.rs

```diff
@@ -9,6 +9,10 @@ pub struct CliArgs {
     #[arg(long, short = 'w', env = "FIL_WALLET_ADDRESS", name = "FIL ADDRESS")]
     pub wallet_address: String,
 
+    /// Unique identifier of the Filecoin Station (required).
+    #[arg(long, short = 's', env = "STATION_ID", name = "STATION ID")]
```

Comment:
I feel the short-hand name `-s` is confusing, e.g. `--state_root` can be abbreviated as `-s` too.

Let's not define any short-hand name for this arg.

```suggestion
    #[arg(long, env = "STATION_ID", name = "STATION ID")]
```